README.md

```diff
@@ -160,6 +160,7 @@ Inspired by the [Awesome lists](https://github.com/sindresorhus/awesome).
 - [App Inventor + IoT](http://iot.appinventor.mit.edu/#/microbit/microbitintro) - Control a micro:bit via Bluetooth with App Inventor, a visual programming environment for Android applications.
 - [BlockyTalkyBLE](http://www.playfulcomputation.group/blockytalkyble.html) - MakeCode and App Inventor extension that makes it easy to connect AppInventor mobile phone apps with the BBC micro:bit wirelessly over Bluetooth.
 - [DroidScript - MicroBit Plugin](https://play.google.com/store/apps/details?id=org.droidscript.microbit) - Allows you you to control the BBC micro:bit remotely from your own DroidScript apps (Android apps written in JavaScript).
+- [CBMicroBit](https://github.com/Louismac/CBMicroBit) - A CoreBluetooth wrapper that connects a Micro:Bit to a computer running OSX using BLE and outputs over OSC (standalone, C++ and Objective C libraries)
```

Comment:
If you could change micro:bit to all lower case, and a full stop at the end then this will be ready for merge.